config/config.go

```diff
@@ -204,8 +208,8 @@ func NewConfig() (c *Config) {
 	c.Server.Address = strings.TrimSpace(getDefaultString(EnvServerAddress, ""))
 	c.Server.Port = getDefaultInt(EnvServerPort, 20000)
 	c.Server.Credentials = security.Credentials{
-		Username:       getDefaultString(EnvServerCredentialsUsername, ""),
-		Password:       getDefaultString(EnvServerCredentialsPassword, ""),
+		Username:       getDefaultStringFromFile(LoginSecretUsername, ""),
+		Passphrase:     getDefaultStringFromFile(LoginSecretPassphrase, ""),
 		AllowAnonymous: getDefaultBool(EnvServerAllowAnonymousAccess, false),
```

Comment:
re: config/authentication.go - I did not find anywhere that that is used. I think the oauth stuff eliminated the need for it, but the file remained. This PR just deletes it because it seems to be unused. If it is STILL used, let me know and I can put it back. But I can't find out where it is used.

Reply:
It's OK to remove that file. I also cannot find any references.